regenerate.sh

```diff
@@ -73,4 +73,4 @@ rm ${WORKDIR}/out/google.golang.org/grpc/reflection/grpc_testingv3/testv3.pb.go
 # grpc/service_config/service_config.proto does not have a go_package option.
 cp ${WORKDIR}/out/grpc/service_config/service_config.pb.go internal/proto/grpc_service_config
 
-cp -R ${WORKDIR}/out/google.golang.org/grpc/ .
+cp -R ${WORKDIR}/out/google.golang.org/grpc/* .
```

Comment:
Hmm ... is this really required?
Works without the `*` on my macbook.

Reply:
Not on my machine, and not on travis. So this is a difference between Mac and Linux?

Also, only travis cron job regenerates proto, so the failure is here: https://travis-ci.org/github/grpc/grpc-go/jobs/680569382

Reply:
Yes, that is weird. Tried on my linux machine and you are right. The difference is that in linux, it seems to create the parent directory as well when you don't specify `*`